scipy/spatial/_spherical_voronoi.py

```diff
@@ -216,16 +139,10 @@ class SphericalVoronoi:
 
     """
 
-    def __init__(self, points, radius=None, center=None, threshold=1e-06):
+    def __init__(self, points, radius=1, center=np.zeros(3), threshold=1e-06):
```

Comment:
Mutable default argument values are usually best not used, so
```
def __init__(..., center=None, ...)
    if center is None:
        center = np.zeros(3)
```

Reply:
Thanks - I did not know that.